packages/cf-component-label/src/Label.js

```diff
@@ -1,13 +1,35 @@
 import React, { PropTypes } from 'react';
+import { createComponent } from 'cf-style-container';
+
+const styles = props => {
+  const theme = props.theme;
+  return {
+    borderRadius: theme.borderRadius,
+    color: theme.color,
+    display: theme.display,
+    fontSize: theme.fontSize,
+    fontWeight: theme.fontWeight,
+    lineHeight: theme.lineHeight,
+    paddingTop: theme.paddingTop,
+    paddingRight: theme.paddingRight,
+    paddingBottom: theme.paddingBottom,
+    paddingLeft: theme.paddingLeft,
+    textTransform: theme.textTransform,
+    userSelect: theme.userSelect,
+    verticalAlign: theme.verticalAlign,
+    '-webkit-text-stroke': theme.webkitTextStroke,
+    whiteSpace: theme.whiteSpace,
+    backgroundColor: theme[`${props.type}BackgroundColor`]
+  };
+};
 
 class Label extends React.Component {
   render() {
-    return React.createElement(
-      this.props.tagName,
```

Comment:
The reason is because I checked our codebase and the only implementation is using span. This also breaks our pattern of having a standardized component, a user could technically render anything using the tagName.